site/content/zh-CN/docs/tasks/run/jobs.md

```diff
@@ -1,96 +1,95 @@
 ---
-title: "Run A Kubernetes Job"
-linkTitle: "Kubernetes Jobs"
+title: "运行 Kubernetes 作业"
+linkTitle: "Kubernetes 作业"
 date: 2022-02-14
 weight: 4
 description: >
-  Run a Job in a Kubernetes cluster with Kueue enabled.
+  在启用了 Kueue 的 Kubernetes 集群中运行作业。
```

Comment:
```suggestion
  在启用了 Kueue 的 Kubernetes 集群中运行 Job。
```
When `Job` is capitalized, it's better to keep it original and do not translate it.